examples/text-generation/run_generation.py

```diff
@@ -211,9 +211,7 @@ def setup_parser(parser):
         action="store_true",
         help="Store kv-cache in float8 when kv-cache is used",
     )
-    parser.add_argument(
-        "--enable_synapse_quantization", "-esq", action="store_true", help="Enable Quantization to fp8"
-    )
+    parser.add_argument("--fp8", action="store_true", help="Enable Quantization to fp8")
```

Comment:
I think the flag name is misleading,  it seems like a data type and not like different flow,
also in future we may be able to quantize to other data types